### PR TITLE
[bitnami/kube-prometheus] Allow to specify probe paths in the kube-prometheus chart (#6780)

### LINKED ISSUE #6780: [bitnami/kube-prometheus] Readiness probe and liveness probe fail when setting routePrefix
**Which chart**:
kube-prometheus

**Describe the bug**
When setting a value for routePrefix, such as /prometheus1, the readiness probe and liveness probes use a path without routePrefix and fail. It is not possible to configure the probe paths, since the values are hardcoded in the chart template.

**To Reproduce**

1. Set routePrefix with /prometheus1
2. Set livenessProbe.enabled: true and readinessProbe.enabled: true
4. The pod fails to reach the running state cause the probes.

**Expected behavior**
At least, It should be possible to change the probe paths.
Better the probe paths is inferred with the routePath.

**Version of Helm and Kubernetes**:

```
version.BuildInfo{Version:"v3.5.1", GitCommit:"32c22239423b3b4ba6706d450bd044baffdcf9e6", GitTreeState:"clean", GoVersion:"go1.15.7"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.1", GitCommit:"c4d752765b3bbac2237bf87cf0b1c2e307844666", GitTreeState:"clean", BuildDate:"2020-12-18T12:09:25Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.5", GitCommit:"6b1d87acf3c8253c123756b9e61dac642678305f", GitTreeState:"clean", BuildDate:"2021-03-18T01:02:01Z", GoVersion:"go1.15.8", Compiler:"gc", Platform:"linux/amd64"}
```

**Additional context**
None

### PR DESCRIPTION
**Pull request by ecureuil**
**Description of the change**

Adding customizable values in the chart of kube-prometheus: the probe paths is no more hardcoded in template.

More flexible configuration and help to solve the issue #6780 when setting the routePath value.

  - fixes #6780 

**Additional information**

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks so much for the contribution @ecureuil ! The changes look fine to me but could you please bump the minor version in the **Chart.yaml**? Thanks in advance.

**Comment by dajose** [comment]
I wonder if we could reuse the routePrefix instead of adding a new parameter

**Comment by ecureuil** [comment]
> 
> 
> I wonder if we could reuse the routePrefix instead of adding a new parameter

It could be a good thing to check consistency. However I think this attribute should be parameterized for flexibility. This is the choice made in a lot of charts to allow the setting of liveness and readiness paths.

File: bitnami/kube-prometheus/Chart.yaml [hunk 1]
```diff
@@ -34,4 +34,4 @@ sources:
   - https://github.com/bitnami/bitnami-docker-prometheus
   - https://github.com/bitnami/bitnami-docker-alertmanager
   - https://github.com/prometheus-operator/kube-prometheus
-version: 6.0.1
+version: 6.0.2
```
Comment (marcosbc) [inline bitnami/kube-prometheus/Chart.yaml:37]: Since this is adding a new feature, we should increase the minor version (i.e. `6.0.1` -> `6.1.0`)

**Review by marcosbc — CHANGES_REQUESTED** [review]
Thanks for the contribution! Could you have a look at my comments?

File: bitnami/kube-prometheus/Chart.yaml (hunk 1 above)
Comment (ecureuil) [reply bitnami/kube-prometheus/Chart.yaml]: You are right, it's fixed.

**Review by marcosbc — APPROVED** [review]
LGTM, thanks!

**Pull request merged**